m/configDataPreprocessingSeaexplorer.m

```diff
@@ -0,0 +1,129 @@
+  preprocessing_options.heading_list(1).conversion = @deg2rad;
+
+% $$$   preprocessing_options.time_list(2).time = 'Posixtime_nav';  
+% $$$   preprocessing_options.position_list(1).longitude = 'Lon';
+% $$$   preprocessing_options.position_list(1).latitude  = 'Lat';
+% $$$   preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  
+  %% CTD (each serie is a different configuration of glider sensors)
+  %  **All variables from a serie must be found to go forward**
+  preprocessing_options.ctd_list(1).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(1).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(1).pressure     = 'GPCTD_PRESSURE';
+  preprocessing_options.ctd_list(1).oxygen       = 'GPCTD_DOF';
+
+  preprocessing_options.ctd_list(2).conductivity = 'SBD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(2).temperature  = 'SBD_TEMPERATURE';
+  preprocessing_options.ctd_list(2).pressure     = 'SBD_PRESSURE';
+  preprocessing_options.ctd_list(2).oxygen       = 'SBD_DOF';
+
+  preprocessing_options.ctd_list(3).conductivity = 'GPCTD_CONDUCTIVITY';
+  preprocessing_options.ctd_list(3).temperature  = 'GPCTD_TEMPERATURE';
+  preprocessing_options.ctd_list(3).pressure     = 'GPCTD_PRESSURE';
+
+  %% Optics
+  preprocessing_options.optics_list(1).chlorophyll = 'FLBBCD_CHL_SCALED';
+  preprocessing_options.optics_list(1).cdom        = 'FLBBCD_CDOM_SCALED';
+  preprocessing_options.optics_list(1).backscatter_700 = 'FLBBCD_BB_700_SCALED';
+
+  preprocessing_options.optics_list(2).chlorophyll = 'TRI_CHL_SCALED';
+  preprocessing_options.optics_list(2).cdom        = 'TRI_CDOM_SCALED';
+  preprocessing_options.optics_list(2).backscatter_700 = 'TRI_BB_700_SCALED';
+
+  preprocessing_options.optics_list(3).chlorophyll = 'FLNTU_CHL_SCALED';
+  preprocessing_options.optics_list(3).turbidity = 'FLNTU_NTU_SCALED';  
+  
+  
+  %% MFL
+  preprocessing_options.mfl_list(1).mfl_v1 = 'MFL_V1';
+  preprocessing_options.mfl_list(1).mfl_v2 = 'MFL_V2';
+  preprocessing_options.mfl_list(1).mfl_v3 = 'MFL_V3';
+  preprocessing_options.mfl_list(1).mfl_v4 = 'MFL_V4';
+  preprocessing_options.mfl_list(1).mfl_te = 'MFL_TMPE';
+  preprocessing_options.mfl_list(1).mfl_td = 'MFL_TMPD';  
+
+  preprocessing_options.mfl_list(2).mfl_v1 = 'UV1_V1';
+  preprocessing_options.mfl_list(2).mfl_v2 = 'UV1_V2';
+  preprocessing_options.mfl_list(2).mfl_v3 = 'UV1_V3';
+  preprocessing_options.mfl_list(2).mfl_v4 = 'UV1_V4';  
+  preprocessing_options.mfl_list(2).mfl_te = 'UV1_TMP1';
+  preprocessing_options.mfl_list(2).mfl_td = 'UV1_TMP2';  
+
+  preprocessing_options.mfl_list(3).mfl_v1 = 'UV1_PHE';
+  preprocessing_options.mfl_list(3).mfl_v2 = 'UV1_TRY';
+  preprocessing_options.mfl_list(3).mfl_v3 = 'UV1_LD1';
+  preprocessing_options.mfl_list(3).mfl_v4 = 'UV1_LD2';
+  preprocessing_options.mfl_list(3).mfl_te = 'UV1_TMP1';
+  preprocessing_options.mfl_list(3).mfl_td = 'UV1_TMP2';  
+
```

Comment:
Do you mean that a given set of raw measurements from the device
can be converted to different physical quantity (concentrations of one or another hydrocarbon)
just by applying a different calibration equation to the same raw measurements?
Or that you can select a device configuration adjusted to measure one of those quantities?
The later is the case for the already handled optic sensors like the WetLabs ECO Triplet.
The calibration is done in the preprocessing,
calling the calibration function set in the preprocessing configuration,
with the calibration parameters in the deployment information.
Thus, the final output is data in engineering units, ready for user consumption.
In fact, this the main goal of the toolbox: turn raw data in glider-model-specific format
into self-describing scientific datasets ready to interpret.
If done otherwise, the final datasets are useless because they can not be interpreted
without additional information.